packages/fxa-auth-server/lib/oauth/db/index.js

```diff
@@ -31,10 +31,15 @@ class OauthDB {
       await preClients();
       await scopes();
     });
-    this.redis = redis(
+    this.redisAccessTokens = redis(
       { ...config.get('redis.accessTokens'), enabled: true },
       logger
     );
+    this.redisRefreshTokens = redis(
+      { ...config.get('redis.refreshTokens') },
+      logger
+    );
+
```

Comment:
@dannycoates seeing this next to your work on storing the access tokens in redis, does it still make sense to store the refresh-tokens stuff in a different redis instance (the one with the sessiontokens) or would it be cleaner to re-use the redis it's already got from your PR?  I recall we may have wanted to keep the access-tokens one separate for operational reasons.

Reply:
Maybe it makes sense to have the line mirror the auth/oauth mysql db split? I'm not thinking of any ways having this metadata alongside access tokens would be a security issue.

Reply:
> Maybe it makes sense to have the line mirror the auth/oauth mysql db split? 

Not 100% sure what you mean here; is it "there's an auth mysql + auth redis, and there's an oauth mysql + oauth redis"? That makes sense to me FWIW.

Reply:
That’s what I meant 😀

Reply:
So this turns out to be kind of awkward because of the way the redis abstractions sets a key prefix as though it's a conection-level settings, but I think I found a way to encapsulate that in an acceptable fashion. We'll see how it shakes out...